backend/apps/common/serializers.py

```
@@ -97,3 +99,32 @@ def update(self, instance, validated_data):
+    email = serializers.EmailField(
+        required=False,
+        allow_blank=False,
+        error_messages={
+            "blank": "El correo electrónico no puede estar vacío.",
+            "invalid": "Por favor, introduce un correo válido."
+        },
+    )
+
+    class Meta:
+        model = UserModel
+        fields = ["first_name", "last_name", "username", "email"]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)